tests/TestCase/Controller/Component/CommonComponentTest.php

```diff
@@ -315,4 +316,45 @@ public function testTrimData() {
 		$this->assertSame($expected, $pass);
 	}
 
+	/**
+	 * @return void
+	 */
+	#[\PHPUnit\Framework\Attributes\DoesNotPerformAssertions]
+	public function testAllowExtensionNone(): void {
+		$this->Controller->Common->allowExtension('');
+	}
+
+	/**
+	 * @return void
+	 */
+	public function testAllowExtensionFail(): void {
+		$this->expectException(NotFoundException::class);
+
+		$this->Controller->Common->allowExtension('csv');
+	}
+
+	/**
+	 * @return void
+	 */
+	#[\PHPUnit\Framework\Attributes\DoesNotPerformAssertions]
+	public function testAllowExtension(): void {
+		$request = $this->Controller->getRequest();
+		$request = $request->withParam('_ext', 'csv');
+		$this->Controller->setRequest($request);
+		$this->Controller->Common->allowExtension('csv');
+	}
+
+	/**
+	 * @return void
+	 */
+	public function testAllowExtensionWrongOnes(): void {
+		$request = $this->Controller->getRequest();
+		$request = $request->withParam('_ext', 'csv');
+		$this->Controller->setRequest($request);
+
+		$this->expectException(NotFoundException::class);
+
+		$this->Controller->Common->allowExtension(['pdf', 'xml']);
+	}
+
```

Comment:
Theoretically, this case wasn't covered, yet:
```suggestion

	/***
	 * Test allowing more than one extensions in URL, of which one was used
     *
	 * @return void
	 */
	#[\PHPUnit\Framework\Attributes\DoesNotPerformAssertions]
	public function testAllowExtensions(): void {
		$request = $this->Controller->getRequest();
		$request = $request->withParam('_ext', 'csv');
		$this->Controller->setRequest($request);
		$this->Controller->Common->allowExtension('csv', 'xml');
	}

```